src/layouts/index.js

```diff
@@ -0,0 +1,137 @@
+import React from 'react'
+import Helmet from 'react-helmet'
+import Header from '../components/Header'
+
+import '../static/styles/index.scss';
+import 'prismjs/themes/prism-solarizedlight.css';
+
+import SideBar from "../components/SideBar";
+import AboutSideBar from "../components/AboutSideBar";
+import Footer from "../components/Footer";
+
+export default class Template extends React.Component {
+
+  render() {
+    if (this.props.location.pathname === '/') {
+      return (
+        <div>
+          <Helmet
+            title="FrintJS Documentation"
+            meta={[
+              {name: 'description', content: 'FrintJS - The Modular JavaScript Framework'},
+              {name: 'keywords', content: 'frint'},
+            ]}
+          />
+
+          {this.props.children()}
+
+          <Footer/>
+
+        </div>
+      )
+    } else if (this.props.location.pathname === '/repl' || this.props.location.pathname === '/repl/') {
+      return (
+        <div>
+          <Helmet
+            title="REPL | Frint"
+            meta={[
+              {name: 'description', content: 'FrintJS - The Modular JavaScript Framework'},
+              {name: 'keywords', content: 'frint'},
+            ]}
+          />
+          <div className="page-repl">
+            <nav className="nav main-nav">
+              <Header/>
+            </nav>
+
+            {this.props.children()}
+          </div>
+          <Footer/>
+
+        </div>
+      )
+    } else if (this.props.location.pathname === '/about' || this.props.location.pathname === '/about/') {
```

Comment:
It might be easier to extract `pathname` and ensure it doesn't have a trailing slash, instead of checking for two values in every `else if`